src/um/winnt.rs

```diff
@@ -968,7 +968,7 @@ STRUCT!{struct XSTATE_CONTEXT {
     Area: PXSAVE_AREA,
     Buffer: PVOID,
 }}
-#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+#[cfg(any(target_pointer_width = "32", target_pointer_width = "64"))]
 IFDEF!{
```

Comment:
This cfg and `IFDEF!` aren't needed as the contents appear to be defined unconditionally in the headers.

Reply:
agree..